Create Blog Post “post-hackathon-the-e-learning-app-built-with-video-api” (#1904)
Nexmo/deved-platform: content/blog/en/post-hackathon-the-e-learning-app-built-with-video-api.md

---
title: "Post-hackathon: The E-learning App Built With Video API"
description: Weeks after the hackathon, the Tokyo Bootcamp students built
  another innovative project with the Video API.
author: clarisse-ng
published: true
published_at: 2021-12-07T09:52:07.319Z
updated_at: 2021-12-07T09:52:07.359Z
category: event
tags:
  - video-api
  - hackathon
  - react
comments: true
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---
# Introduction

> In our previous post about [Vonage Joins Code Chrysalis API Hackathon](https://learn.vonage.com/blog/2021/09/30/vonage-joins-code-chrysalis-api-night-hackathon/), we saw how Team Subarshi built an application called [CollabSpace](https://vonage-hackathon.herokuapp.com/) using [Vonage Video API](https://www.vonage.com/communications-apis/video/) to make online team collaboration more interactive and simple for connecting with others and editing documents in real-time. That was all, right? Of course not. 

Reviewer: ```suggestion
In our previous post about [Vonage Joins Code Chrysalis API Hackathon](https://learn.vonage.com/blog/2021/09/30/vonage-joins-code-chrysalis-api-night-hackathon/), we saw how Team Subarashi built an application called [CollabSpace](https://vonage-hackathon.herokuapp.com/) using [Vonage Video API](https://www.vonage.com/communications-apis/video/) to make online team collaboration more interactive and simple for connecting with others and editing documents in real-time. That was all, right? Of course not. 
```

Reviewer 2: Added onto Amanda's comment to shorten sentences and hopefully improve clarity?

Reviewer 3: Yes it does, thank you @garann and @amdcavallaro

> In our previous post about [Vonage Joins Code Chrysalis API Hackathon](https://learn.vonage.com/blog/2021/09/30/vonage-joins-code-chrysalis-api-night-hackathon/), we saw how Team Subarshi built an application called [CollabSpace](https://vonage-hackathon.herokuapp.com/) using [Vonage Video API](https://www.vonage.com/communications-apis/video/) to make online team collaboration more interactive and simple for connecting with others and editing documents in real-time. That was all, right? Of course not. 

Reviewer: ```suggestion
In our previous post about [Vonage Joins Code Chrysalis API Hackathon](https://learn.vonage.com/blog/2021/09/30/vonage-joins-code-chrysalis-api-night-hackathon/), we saw how Team Subarashi built an application called [CollabSpace](https://vonage-hackathon.herokuapp.com/) using [Vonage Video API](https://www.vonage.com/communications-apis/video/) to make online team collaboration more interactive and simple for connecting with others and editing documents in real-time. That was all, right? Of course not. 
```

> Within a few weeks following the hackathon, this group of Bootcamp students used the [Video API](https://www.vonage.com/communications-apis/video/) to build another innovative project which was an e-learning web application called [Senpai](https://senpai-container-flsg4ziguq-uc.a.run.app/#/), for their Bootcamp Demo Day. In Japanese, the word Senpai refers to a teacher or master.

Reviewer 2: ```suggestion
Within a few weeks following the hackathon, this group of Bootcamp students used the [Video API](https://www.vonage.com/communications-apis/video/) to build another innovative project, an e-learning web application called [Senpai](https://senpai-container-flsg4ziguq-uc.a.run.app/#/), for their Bootcamp Demo Day. In Japanese, the word _senpai_ refers to a teacher or master.
```

> # What did they build?

Reviewer 2: ```suggestion
## What Did They Build?
```

![Home page of the Senpai, e-learning web application ](/content/blog/post-hackathon-the-e-learning-app-built-with-video-api/senpai.png "Home page of the Senpai, e-learning web application")

> Senpai

Reviewer 2: ```suggestion

```

This seems to be a typo or cut-and-paste error?

> [Senpai](https://senpai-container-flsg4ziguq-uc.a.run.app/#/) is an e-learning web application that connects programming mentors and mentees. The website allows users interested in learning programming skills to find experienced front-end developers for 1-on-1 tutoring. It could also provide tutors with an opportunity to earn money as freelancers.

Reviewer 2: ```suggestion
[Senpai](https://senpai-container-flsg4ziguq-uc.a.run.app/#/) is an e-learning web application that connects programming mentors and mentees. The website allows users interested in learning programming skills to find experienced front-end developers for one-on-one tutoring. It could also provide tutors with an opportunity to earn money as freelancers.
```

Built with:

* MongoDB
* Express
* React
* Nodejs
* Docker
* [Vonage Video API](https://tokbox.com/developer/)

You can check out their [Application](https://senpai-container-flsg4ziguq-uc.a.run.app/#/) and see their code on [GitHub](https://github.com/Team-Subarashi/senpai). You can also view their [demo and presentation](https://www.youtube.com/watch?v=O5HU8BrGvJQ).

![In the “Find a Senpai” page, you can see a list of Senpai when you set the filter to All](/content/blog/post-hackathon-the-e-learning-app-built-with-video-api/find-a-senpai.png "In the “Find a Senpai” page, you can see a list of Senpai when you set the filter to All")

> # What makes it so special?

Reviewer 2: ```suggestion
## What Makes It So Special?
```

> The idea behind Senpai was to create a platform for programmers to provide 1-on-1 tutorial services for people wanting to learn frontend technologies. The app was designed so that users would not have to constantly switch between multiple apps throughout the process, including those for scheduling, video chat, and screen sharing.

Reviewer 2: ```suggestion
The idea behind Senpai was to create a platform for programmers to provide one-on-one tutorial services for people wanting to learn frontend technologies. The app was designed so that users would not have to constantly switch between multiple apps throughout the process, including those for scheduling, video chat, and screen sharing.
```

> # Conclusion

Reviewer 2: ```suggestion
## Conclusion
```

> I thought their project was a brilliant idea! I was impressed by what they have created using the [Video API](https://tokbox.com/developer/). Within a short time, they mastered the API and came up with a solution that streamlined the process of scheduling appointments, connecting people through video chat, screen sharing, and collaborating with one another all in one place. I can almost see how it can also be used in a virtual classroom, tuition centers, and distant learning in international schools.

Reviewer 2: ```suggestion
I thought their project was a brilliant idea! I was impressed by what they have created using the [Video API](https://tokbox.com/developer/). Within a short time, they mastered the API and came up with a solution that streamlined the process of scheduling appointments, connecting people through video chat, screen sharing, and collaborating with one another all in one place. I can almost see how it can also be used in virtual classrooms or tuition centers, and for distance learning in international schools.
```

> # Want to know more?

Reviewer 2: ```suggestion
## Want To Know More?
```

> You can learn more about our [Video API](https://www.vonage.com/communications-apis/video/) and how the [video platform](https://tokbox.com/developer/guides/basics/) works. Now, it is your turn to [get started](https://tokbox.com/developer/) build a project, or try out the [Video Express](https://tokbox.com/developer/video-express/) to create a multiparty video conferencing web application, without video expertise.

Reviewer 2: ```suggestion
You can learn more about our [Video API](https://www.vonage.com/communications-apis/video/) and how the [video platform](https://tokbox.com/developer/guides/basics/) works. Now it is your turn to [get started](https://tokbox.com/developer/) building a project, or to try out [Video Express](https://tokbox.com/developer/video-express/) to create a multiparty video conferencing web application, without video expertise.
```

> # Show us what you are working on

Reviewer 2: ```suggestion
## Show Us What You Are Working On
```
